.github/workflows/run-terraform.yml

```
@@ -113,7 +110,22 @@ env:
   TF_OPTION_3: ${{ inputs.terraform_option_3 }}
 
 jobs:
+  setup-env:
+    runs-on: ubuntu-latest
+    outputs:
+      workload_identity_provider: ${{ steps.set-output.outputs.WORKLOAD_IDENTITY_PROVIDER }}
+    steps:
+      - name: Set vars
+        id: set-output
+        run: |
+          PRODUCT_NAME=$(echo ${{ env.SERVICE_ACCOUNT }} | sed 's/-.*//')
+          DEFAULT_WORKLOAD_IDENTITY="projects/${{ env.AUTH_PROJECT_NUMBER }}/locations/global/workloadIdentityPools/${PRODUCT_NAME}-deploy-pool/providers/github-provider"
+          OVERRIDE=${{ env.WORKLOAD_IDENTITY_PROVIDER_OVERRIDE }}
```

Ser ut som dere kan hente rett fra env med bash her uten å bruke github context?

```suggestion
          PRODUCT_NAME=$(echo $SERVICE_ACCOUNT | sed 's/-.*//')
          DEFAULT_WORKLOAD_IDENTITY="projects/$AUTH_PROJECT_NUMBER/locations/global/workloadIdentityPools/$PRODUCT_NAME-deploy-pool/providers/github-provider"
          OVERRIDE=$WORKLOAD_IDENTITY_PROVIDER_OVERRIDE
```